internal/transfers/repository.go

```diff
@@ -60,15 +60,17 @@ func (r *SQLRepo) Close() error {
 	return r.db.Close()
 }
 
-func (r *SQLRepo) getUserTransfers(userID id.User) ([]*model.Transfer, error) {
-	query := `select transfer_id from transfers where user_id = ? and deleted_at is null`
+func (r *SQLRepo) getUserTransfers(userID id.User, params transferFilterParams) ([]*model.Transfer, error) {
+	query := `select transfer_id from transfers
+where user_id = ? and created_at >= ? and created_at <= ? and deleted_at is null
```

Comment:
Do we have an index for `created_at` thats `desc` ?

Reply:
Given that its `X or equal to` Is it fine to get duplicates if they query this by sequential dates?

Reply:
There's no index on `created_at desc` for transfers. We should add one... Thanks for the catch